sql/src/test/java/io/crate/integrationtests/SysNodeResiliencyIntegrationTest.java

```diff
@@ -77,4 +76,16 @@ public void testTimingOutNode() throws Exception {
             waitNoPendingTasksOnAll();
         }
     }
+
+    /**
+     * Test cluster name check, when path.data does not contain cluster
+     * name as a folder.
+     *
+     * TODO: Remove this when ES >= 6.0
+     */
+    @Test
+    public void testClusterNameInPathCheck() throws Exception {
```

Comment:
maybe make the test method more descriptive - e.g.  `testClusterNameInPathCheckDoesNotFailWithDefaultConfig`